src/lib/websocket.ts

```diff
@@ -26,6 +62,23 @@ const wsReceiveSchema = z.union([
       targetId: z.string(),
     }),
   }),
+  z.object({
+    type: z.literal('railMerged'),
+    body: z.object({
+      childId: z.string(),
+      parentId: z.string(),
+      playerId: z.string(),
+    }),
+  }),
+  z.object({
+    type: z.literal('railBlockCreated'),
```

Comment:
バックエンド変えてもいいけどとりあえずは
```suggestion
    type: z.literal('blockCreated'),
```